File: FaceSample/app/src/main/java/com/regula/facesample/items/basic/MatchFacesRequestItem.java
package com.regula.facesample.items.basic;

import android.content.Context;
import android.content.Intent;

import com.regula.facesample.MatchFacesActivity;
import com.regula.facesample.items.CategoryItem;

/**
 * Created by Sergey Yakimchik on 20.08.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class MatchFacesRequestItem extends CategoryItem {

    @Override
    public void onItemSelected(Context context) {
        Intent intent = new Intent(context, MatchFacesActivity.class);
        context.startActivity(intent);
    }

    @Override
    public String getTitle() {
        return "MatchFacesRequest";
    }

    @Override
    public String getDescription() {
        return "Checks the likelihood that two faces belong to the same person.";
    }
}

File: BasicSample/app/src/main/java/com/regula/facesamplekotlin/items/basic/MatchFacesRequestItem.kt
package com.regula.facesamplekotlin.items.basic

import android.content.Context
import android.content.Intent
import com.regula.facesamplekotlin.MatchFacesActivity
import com.regula.facesamplekotlin.category.CategoryItem

class MatchFacesRequestItem : CategoryItem() {

    override fun onItemSelected(context: Context) {
        val intent = Intent(context, MatchFacesActivity::class.java)
        context.startActivity(intent)
    }

    override val title: String
        get() =  "MatchFacesRequest"

    override val description: String
        get() = "Checks the likelihood that two faces belong to the same person"

}